integrated_channels/canvas/client.py

```diff
@@ -221,8 +221,9 @@ def _update_course_details(self, course_id, serialized_data):
                     self.enterprise_configuration.canvas_base_url,
                     course_id,
                 )
+                # Providing the param `event` and setting it to `offer` is equivalent to publishing the course.
```

Comment:
this looks good for now great you found this! we can always re eval if autopublish will be on/off later.